frontend/src/assets/css/progress-bar.css

```diff
@@ -0,0 +1,3 @@
+.progress-bar{
+
+}
```

Comment:
NIT: Missing new line at the end of file.
Although I can see that this is here as a placeholder, so you can leave it.